keyboards/converter/a1200/keymaps/default/keymap.c

```diff
@@ -30,7 +30,7 @@ const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
      KC_ESC,    KC_F1, KC_F2, KC_F3, KC_F4, KC_F5,      KC_F6, KC_F7, KC_F8, KC_F11, KC_F12, 
```

Comment:
You might want to set the keys that are the same on this layer to `_______` (aka `KC_TRNS`) for clarity and readability.

Reply:
Yes, I'll do that.